lib/core/sync_job_runner.rb

```diff
@@ -19,16 +19,14 @@ def initialize(service_type, expected_fields, actual_fields)
   end
 
   class SyncJobRunner
+    JOB_REPORTING_INTERVAL = 10
```

Comment:
As suggested by @sphilipse here: https://github.com/elastic/enterprise-search-team/issues/3198#issuecomment-1305361746, will just make a hardcoded variable in 8.6